excalibur/ancillary/estimators.py

```diff
@@ -1,6 +1,6 @@
 '''estimators ds'''
 
-# Heritage code shame:
```

Comment:
Should leave these comments as are because they are easily searched. Doubt anyone is going to have a comment like it.